liquibase-core/src/main/java/liquibase/change/core/RawSQLChange.java

```diff
@@ -10,29 +10,11 @@
 import liquibase.util.StringUtil;
 
 /**
- * Allows execution of arbitrary SQL.  This change can be used when existing changes are either don't exist,
+ * Allows execution of arbitrary SQL. This change can be used when existing changes are either don't exist,
```

Comment:
"This change can be used when existing changetypes don't exist or are not flexible enough." I'd drop the "or buggy," too.